CONTRIBUTING.md

```diff
@@ -345,6 +345,18 @@ with clear instructions on how to install the required libraries.
 Furthermore, if your package requires CGO, it MUST be able to compile and operate in a no op mode
 or report a warning back to the collector with a clear error saying CGO is required to work.
 
+### Breaking changes
+
+Whenever possible, we adhere to semver as our minimum standards. Even before v1, we strive to not break compatibility
+without a good reason. Hence, when a change is known to cause a breaking change, it should be clearly marked in the
+changelog, possibly with a line instructing users how to move forward.
+
+We also strive to perform breaking changes in two stages, or using feature flags: 
+
+- when we need to remove something, we mark a feature as deprecated in one version, and remove it in the next one
+- when renaming types, functions, or attributes, we create the new name first, deprecating the old one in one
```

Comment:
Maybe we should we explicitly allow moving things across packages and package renames? Otherwise it'll be very difficult to work towards Collector 1.0 given amount of changes we might need to do based on the today's SIG discussions.

Reply:
If we move types or functions to a new package we should be able to leave behind a forwarding type or function.  If we can't do that because it would introduce circular dependencies then we do it the other way around and have the new type/function reference the prior location until it is removed and the definition moved to the new location.

Reply:
Yes, but it still adds a lot of work for us to maintain this approach while this kind of changes is a simple package rename on the client side. I would vote for allowing these changes, let's see what other folks think

Reply:
pdata is mostly generated code.  We can generate it twice in different packages.  

I think that at this point in the lifecycle of the project it is worthwhile to consider that adding burdens on those proposing making breaking changes is a good thing.  It should perhaps no longer be the case that we make breaking changes in an offhand manner because it is easy to do.

Reply:
Should I add something to the PR to address this?

Reply:
I think it is sufficiently covered.